### PR TITLE
[WIP] Copying output locally and running

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: src/startupscriptgenerator/dotnetcore/main.go [hunk 1]
```diff
@@ -31,7 +32,13 @@ func main() {
 	defaultAppFilePathPtr := flag.String(
 		"defaultAppFilePath",
 		"",
-		"[Optional] Path to a default dll that will be executed if the entrypoint is not found. Ex: '/opt/startup/aspnetcoredefaultapp.dll'")
+		"[Optional] Path to a default dll that will be executed if the entrypoint is not found." +
+		" Ex: '/opt/startup/aspnetcoredefaultapp.dll'")
+	copyOutputToDifferentDirAndRunPtr := flag.String(
+		"copyOutputToDifferentDirAndRun",
+		"true",
```
Comment (kichalla) [inline src/startupscriptgenerator/dotnetcore/main.go]: I made this `true` the default just to make sure all the tests pass under this flag.

File: src/startupscriptgenerator/dotnetcore/main.go [hunk 2]
```diff
@@ -48,6 +55,30 @@ func main() {
 		fullDefaultAppFilePath = common.GetValidatedFullPath(*defaultAppFilePathPtr)
 	}
 
+	if (*copyOutputToDifferentDirAndRunPtr == "true") {
```
Comment (kichalla) [inline src/startupscriptgenerator/dotnetcore/main.go]: Idea here was to make code changes as less intrusive as possible, so once the copy is done and the paths have been modified appropriately, rest of the existing code should work as it was before

File: src/startupscriptgenerator/dotnetcore/main.go [hunk 3]
```diff
@@ -48,6 +56,48 @@ func main() {
...
+		srcFolder := fullPublishedOutputPath
+		if (srcFolder == "") {
+			// The output folder is a sub-directory of this source directory
+			srcFolder = fullSourcePath
+		}
+
+		scriptPath := "/tmp/test.sh"
```
Comment (gpcastro) [inline src/startupscriptgenerator/dotnetcore/main.go]: are you thinking of making this a separate script, or part of the main one for running the app?

**Comment by kichalla** [comment]
I will be using the other WIP PR further

**Pull request closed without merging**